primefaces-selenium/primefaces-selenium-core/src/main/java/org/primefaces/selenium/AbstractPrimePageTest.java

```diff
@@ -192,6 +195,17 @@ protected void assertNoJavascriptErrors() {
         Assertions.assertTrue(severe.isEmpty(), "Javascript errors were detected in the browser console.\r\n" + severe.toString());
     }
 
+    /**
+     * Clears the browser console.
+     */
+    protected void clearConsole() {
+        // https://stackoverflow.com/questions/51404360/how-to-clear-console-errors-using-selenium
+        JavascriptExecutor js = (JavascriptExecutor) getWebDriver();
```

Comment:
@christophs78 just one thing: dont we have something like PrimeSelenium.executeScript?